static/js/reducers/index.js

```diff
@@ -63,6 +68,46 @@ export const ui = (state = INITIAL_UI_STATE, action) => {
   }
 };
 
+export const INITIAL_DIALOG_STATE = {
+  openDialog: false,
+  degreeLevel: '',
+  educationIndex: null,
+  educationId: null
+};
+export const educationDialog = (state = INITIAL_DIALOG_STATE, action) => {
+  switch (action.type) {
+  case SHOW_EDUCATION_FORM_DIALOG:
+    return Object.assign({}, state, {
+      openDialog: true,
+      degreeLevel: action.payload.level,
+      educationIndex: action.payload.index
+    });
+  case HIDE_EDUCATION_FORM_DIALOG:
+    return INITIAL_DIALOG_STATE;
+  default:
+    return state;
+  }
+};
+let education_level_state = {};
+education_level_state[HIGH_SCHOOL] = true;
+education_level_state[ASSOCIATE] = true;
+education_level_state[BACHELORS] = true;
+education_level_state[MASTERS] = false;
+education_level_state[DOCTORATE] = false;
+
+export const INITIAL_EDUCATION_LEVEL_STATE = education_level_state;
```

Comment:
FYI, `const` only applies to the reference to the variable, not the contents, so you don't need the `education_level_state` here (feel free to leave it though)